src/main/java/net/mcreator/ui/minecraft/states/block/JBlockPropertiesStatesList.java

```diff
@@ -147,7 +152,8 @@ public void setProperties(List<PropertyDataWithValue<?>> properties) {
 	public AggregatedValidationResult getValidationResult() {
 		AggregatedValidationResult validationResult = new AggregatedValidationResult.PASS();
 		for (JBlockPropertiesListEntry entry : propertiesList) {
-			if (nonUserProvidedProperties.get().contains(entry.getPropertyData().getName().replace("CUSTOM:", ""))) {
+			String regName = BlockStatePropertyUtils.propertyRegistryName(entry.getPropertyData());
+			if (nonUserProvidedProperties.get().stream().anyMatch(regName::equalsIgnoreCase)) {

```

Comment:
equalsIgnoreCase looks like it is "masking" some problem. do we really need it? I believe those names should always contain the correct name

Reply:
If the property is external but has no `other/registry_name` defined, the returned name will be upper-cased. How should the list normally react to the cases like that?

Reply:
How can that happen? I have changed tests to fail if such property exists. We require registry_name. This would be considered invalid entry

Reply:
What if a property is removed in a new generator? `VINE_END` should have existed before since it was included in the data list.

Reply:
We generally remove entries so list is 100% filled for the latest supported generator. This was practice for a while now